tests/std/tests/VSO_0000000_allocator_propagation/test.cpp

```diff
@@ -307,7 +462,9 @@ _CONSTEXPR20 bool test_sequence() {
     test_sequence_copy_assign<Sequence, CopyAlloc<int>>(11, 22, 11); // POCCA, non-equal allocators
     test_sequence_copy_assign<Sequence, CopyEqualAlloc<int>>(11, 22, 11); // POCCA, always-equal allocators
 
+#if defined(__clang__) || defined(__EDG__) // TRANSITION, VSO-1888462
     test_sequence_move_ctor<Sequence>();
+#endif // ^^^ no workaround ^^^

```

Comment:
I think in compiler bug case we can still keep this test at runtime.

Reply:
Great point, I'll validate and push.